### PR TITLE
bastbnl patch django2

### PR DESCRIPTION
**Pull request by bastbnl**

### CODE REVIEW

**Comment by jleclanche** [comment]
master only supports Django 2.0+, please update the PR accordingly (there's no need for pre-2.0 doc or code)

File: docs/install.rst [hunk 1]
```diff
@@ -15,15 +15,18 @@ Add `oauth2_provider` to your `INSTALLED_APPS`
...
 
-If you need an OAuth2 provider you'll want to add the following to your urls.py
+Update your projects' `urls.py` to include this when you need an OAuth2 provider:
 
 .. code-block:: python
 
     urlpatterns = patterns(
         ...
-        url(r'^o/', include('oauth2_provider.urls', namespace='oauth2_provider')),
+        url(r'^o/', include(('oauth2_provider.urls', 'oauth2_provider_app', ), namespace='oauth2_provider'), ),
```
Comment (IvanAnishchuk) [inline docs/install.rst]: There's also this new way of configuring urlpatterns with `path`. Maybe we should mention it here?

Comment (IvanAnishchuk) [reply docs/install.rst]: Also whitespace before closing bracket.

File: oauth2_provider/middleware.py [hunk 2]
```diff
@@ -1,8 +1,18 @@
 from django.contrib.auth import authenticate
 from django.utils.cache import patch_vary_headers
 
+# bastb Django 1.10 has updated Middleware. This code imports the Mixin required to get old-style
```
Comment (IvanAnishchuk) [inline oauth2_provider/middleware.py]: Maybe it wouldn't be too hard to rewrite it the new style?

**Comment by bastbnl** [comment]
Just noticed your comments, @IvanAnishchuk. I'll try to recreate the branch this weekend and create a new pull request

**Comment by n2ygk** [comment]
No followup since 2019. Closing.

**Pull request closed without merging**